frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -126,23 +124,35 @@ class Gallery extends React.Component {
     document.removeEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
-  buildBreadcrumbs = () => {
+  async loadContentDetails () {
+    const { state } = this
+
+    const workspaceLabel = await this.loadWorkspaceData()
```

Comment:
wrong naming. loadWorkspaceData should return an object with a property `label`. Or loadWorkspaceData should be called loadWorkspaceLabel()